extensions/security/runtime/src/main/java/io/quarkus/security/runtime/QuarkusSecurityIdentity.java

```diff
@@ -212,6 +217,10 @@ public Builder addPermissionChecker(Function<Permission, Uni<Boolean>> function)
         }
 
```

Comment:
I would put a setAnonymous(boolean anyonymous) method here. But maybe I need to checkout the code to see why this doesn't work.

Reply:
@renegrob Hey, ha-ha, sorry, I'm a bit all over the place - of course it has to be added here, for some reasons (may be because I was not excited about this option) - I thought it would have to be added in `quarkus-security`, will push an update shortly